go.mod

```diff
@@ -1,9 +1,93 @@
 module github.com/auth0/terraform-provider-auth0
 
-go 1.16
+go 1.18
 
 require (
 	github.com/auth0/go-auth0 v0.6.1
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/terraform-plugin-sdk v1.16.1
 )
+
+require (
```

Comment:
Please check https://go.dev/doc/go1.17#go-command for more details as to why these indirect deps are surfaced within the go.mod instead of keeping them only in the go.sum.

Reply:
**Question (non-blocking):** So if I read this correctly, since we're moving forward with Go 1.18 and presumably most of our dependencies do or will, we can expect this separate require block to always contain these indirect dependencies. Or posed another way, this isn't going away, right?

Reply:
**Question (non-blocking):** So if I read this correctly, since we're moving forward with Go 1.18 and presumably most of our dependencies do or will, we can expect this separate require block to always contain these indirect dependencies. Or posed another way, this isn't going away, right?

Reply:
That's correct, it's here to stay with us indefinitely.